inbox/react/migration-guide.mdx

```diff
@@ -0,0 +1,435 @@
+---
+title: "Migration Guide"
+sidebarTitle: "Migration Guide"
+description: "This guide outlines the key differences between the `@novu/notification-center` package and the new `@novu/react` package. Follow the steps below to migrate your application to the latest version."
+---
+
+## Getting Started
+
+To begin migrating to `@novu/react`, install the package via npm:
+
+```bash
+npm install @novu/react
+```
+
+## Notification Center without Bell Icon
+
+The @novu/react package introduces a flexible way to display notifications as a list without the default bell icon. Utilize the `Inbox` and `Notifications` components to achieve this functionality.
+
+```tsx
+import { Inbox, Notifications } from "@novu/react";
+
+function Novu() {
+  return (
+    <Inbox
+      applicationIdentifier="YOUR_APP_ID"
+      subscriberId="YOUR_SUBSCRIBER_ID"
+    >
+      <Notifications />
+    </Inbox>
+  );
+}
+
+export default Novu;
+```
```

Comment:
@BiswaViraj In all these sections I'm missing the comparison steps, like the old way of doing it with `NotificationCenter` and a new way with the `Inbox`.

Reply:
Will add the old version 👍🏼